lmfdb/lfunctions/Lfunction.py

```diff
@@ -400,16 +400,23 @@ def convert_coefficient(an, base_power_int):
         return res[0], CDF(res[1])
 
     base_power_int = int(coeff_info[0][2:-3])
+    fix = False
     for n, an in enumerate(L.dirichlet_coefficients_arithmetic):
         L.dirichlet_coefficients_arithmetic[n] , L.dirichlet_coefficients[n] =  convert_coefficient(an, base_power_int)
-
+        # checks if we need to fix the Euler factors
+        if is_prime(n) and L.dirichlet_coefficients_arithmetic[n] != 0:
+            if fix:
```

Comment:
I'm confused: this branch of the if statement won't ever be triggered....

Reply:
It will be triggered when the else changes the value of the fix variable

Reply:
Ok, I was being stupid.